feat: add multi-search dropdown in mobile

jeanphilippeds opened the pull request:
Hello,

A small PR to fix https://github.com/c2corg/c2c_ui/issues/2451

With selected tags:
<img src="https://user-images.githubusercontent.com/22117220/200693207-a80010e8-d2fa-4530-a8eb-f00d1875110f.png" width="400" />

With multi-search results:
<img src="https://user-images.githubusercontent.com/22117220/200693239-05a0512e-872b-4740-b7d3-60c39c849946.png" width="400" />

I'm not a big fan of repeating the `filter` icon. Maybe we could put another one.

brunobesson commented on src/views/documents/utils/QueryItems.vue:
```diff
@@ -17,17 +17,12 @@
         <span slot="button" class="button is-size-7-mobile" :disabled="category.fields.length === 0">
           <fa-icon :icon="$options.categoryIcon[category.name]" />
           <span class="is-hidden-mobile">
-            <!-- $gettext('General') -->
-            <!-- $gettext('ratings') -->
-            <!-- $gettext('Terrain') -->
-            <!-- $gettext('Miscs') -->
```
Is thee a specific reason for removing these comments? They are i18n-related, and should not be removed (except if unused). See https://github.com/c2corg/c2c_ui/wiki/i18n

jeanphilippeds replied:
Ok my bad, I thought it was dead code

brunobesson reviewed (approved):
Other than the removed i18n stuff, looks good to me.

brunobesson commented:
There might be some problems when the criteria title is larger than the phone width:

![image](https://user-images.githubusercontent.com/2234024/200883127-28d72bd7-0ba9-4437-b568-df773b0d7783.png)

jeanphilippeds commented:
Good catch!
I've added some commits. I also fixed the margins between tags when we have multiple lines.

<img src="https://user-images.githubusercontent.com/22117220/200915628-e2b88bde-ad79-48d5-bb2c-fc7a80d9158c.png" width="400" />
<img src="https://user-images.githubusercontent.com/22117220/200915639-c9f925cc-034d-499c-8af9-d13619594642.png" width="400" />

brunobesson commented:
Nice job! I'll merge.
